src/coll/options.rs

```diff
@@ -1,25 +1,39 @@
 //! Options for collection-level operations.
 use bson::{self, Bson};
-use cursor;
 use common::{ReadPreference, WriteConcern};
 use Error::ArgumentError;
 use Result;
 
 /// Describes the type of cursor to return on collection queries.
-#[derive(Clone, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum CursorType {
     NonTailable,
     Tailable,
     TailableAwait,
 }
 
+impl Default for CursorType {
+    fn default() -> Self {
+        CursorType::NonTailable
+    }
+}
+
 /// Describes the type of document to return on write operations.
-#[derive(Clone, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub enum ReturnDocument {
     Before,
     After,
 }
 
+impl ReturnDocument {
```

Comment:
This was moved from below, where it was called `to_bool`. It's more idiomatic to name methods that take `self` as a reference `as_*` and reserve `to_*` for methods that consume self.